PENDING_PENDING_RELEASE_NOTES.md

```diff
@@ -0,0 +1,11 @@
+
+## Version m.n.o
```

Comment:
The "m.n.o" in our documentation is a placeholder for actually filling in the version number. This file will eventually be used in deployment, and the idea is to have the expected version in this file, based on the current RELEASE_NOTES and PENDING_RELEASE_NOTES file versions